setup.py

```diff
@@ -1,70 +1,3 @@
-import os
-import re
+from setuptools import setup
 
-from setuptools import find_packages, setup
-
-# NOTE: If updating requirements make sure to also check Pipfile for any locks
-# NOTE: When updating botocore make sure to update awscli/boto3 versions below
-install_requires = [
-    # pegged to also match items in `extras_require`
-    'botocore>=1.34.70,<1.34.163',
-    'aiohttp>=3.9.2,<4.0.0',
-    'wrapt>=1.10.10, <2.0.0',
-    'aioitertools>=0.5.1,<1.0.0',
-]
-
-extras_require = {
-    'awscli': ['awscli>=1.32.70,<1.33.45'],
-    'boto3': ['boto3>=1.34.70,<1.34.163'],
-}
-
-
-def read(f):
-    return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
-
-
-def read_version():
-    regexp = re.compile(r"^__version__\W*=\W*'([\d.abrc]+)'")
-    init_py = os.path.join(
-        os.path.dirname(__file__), 'aiobotocore', '__init__.py'
-    )
-    with open(init_py) as f:
-        for line in f:
-            match = regexp.match(line)
-            if match is not None:
-                return match.group(1)
-        raise RuntimeError('Cannot find version in aiobotocore/__init__.py')
-
-
-setup(
-    name='aiobotocore',
-    version=read_version(),
-    description='Async client for aws services using botocore and aiohttp',
-    long_description='\n\n'.join((read('README.rst'), read('CHANGES.rst'))),
-    long_description_content_type='text/x-rst',
-    classifiers=[
-        'Development Status :: 4 - Beta',
-        'Intended Audience :: Developers',
-        'Intended Audience :: System Administrators',
-        'Natural Language :: English',
-        'License :: OSI Approved :: Apache Software License',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.8',
-        'Programming Language :: Python :: 3.9',
-        'Programming Language :: Python :: 3.10',
-        'Programming Language :: Python :: 3.11',
-        'Programming Language :: Python :: 3.12',
-        'Environment :: Web Environment',
-        'Framework :: AsyncIO',
-    ],
-    author="Nikolay Novik",
-    author_email="nickolainovik@gmail.com",
-    url='https://github.com/aio-libs/aiobotocore',
-    download_url='https://pypi.python.org/pypi/aiobotocore',
-    license='Apache License 2.0',
```

Comment:
looks like you missed migrating this field

Reply:
I failed to mention it, but I had left it out on purpose, since the license is already specified via trove classifier. But you are completely right: after re-reading specifications, I agree that the license field is not equivalent to the classifier, because it specifies the license version. Will put it back in.

Reply:
That's a separate field that is free-form, and it's displayed separately on PyPI. There's also a PEP in the works where people are trying to bring SPDX in the mix and standardize license specifiers better..